include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -235,6 +235,8 @@ WARNING(cannot_assign_value_to_conditional_compilation_flag,none,
 ERROR(error_optimization_remark_pattern, none, "%0 in '%1'",
       (StringRef, StringRef))
 
+ERROR(error_an_error_occurred,none, "an error occurred", ())
```

Comment:
This is still too vague; imagine how it would look to a user. I still think my phrasing is reasonable, something like "compilation stopped due to errors in other files".

Reply:
How about "error(s) in other file(s) halted compilation"?

Reply:
I feel like the funny pluralization isn't buying us anything, and it also seems weird to me to use active tense for this. My mental model is that the compiler chose to stop compiling *this* file because of errors in *that* file. But maybe others don't share that view.

Reply:
I liked the ambiguous pluralization because it was more accurate. Could be one, could be many errors. But in the interest of time, I can live with the plural. I'll passivate the message, too.